controllers/sensor/validate_test.go

```diff
@@ -61,7 +61,7 @@ func TestValidDependencies(t *testing.T) {
 		err := ValidateSensor(sObj)
 		assert.NotNil(t, err)
 		assert.Equal(t, true, strings.Contains(err.Error(), "more than once"))
-	})
+	})*/
```

Comment:
see comment in controllers/sensor/validate.go (may still want to perform this check but for STAN only...requires changes to validate.go to enable that)